test/api/peer.transactions.main.js

```diff
@@ -77,6 +77,18 @@ describe('POST /peer/transactions', function () {
 			});
 	});
 
+	it('using data field should be ok', function (done) {
+		var account = node.randomAccount();
+		var transaction = node.lisk.transaction.createTransaction(account.address, 1, node.gAccount.password);
```

Comment:
@karmacoma this test case will fail, because here `createTransaction()` from lisk-js package doesn't take into account data field we have added in the backed. So, signature and id will not be generated correctly.